### PR TITLE
Replacement of outmoded terminology and more accurate abstract+motivation 

### PR DESCRIPTION
**Pull request by arjunhassard**

### CODE REVIEW

File: mining-paper.tex [hunk 1]
```diff
@@ -150,47 +146,47 @@ \subsection{Inflation decay}
...
     \label{fig:reward-rate-vs-duration}
 \end{figure}
 
-We want to incentivize miners to serve re-encryption policies for at least $1$~year.
+We want to incentivize stakers to serve re-encryption policies for at least $1$~year.
 However, short-term stakers are still useful and should be compensated.
 We will give the full compensation ($\kappa=1$) to the stakers who are committed to stake at least $T_1=1$~year,
 however those who stake for $T_{\min}=1$~month will get close to half the compensation ($\kappa\approx0.54$) (\figref{fig:reward-rate-vs-duration}).
-The individual daily compensation rate for a miner looks as:
+The individual daily compensation rate for a staker looks as:
```
Comment (derekpierre) [inline mining-paper.tex]: "looks as" reads weirdly.

```suggestion
The individual daily compensation rate for a staker is:
```

File: mining-paper.tex [hunk 2]
```diff
@@ -202,22 +198,22 @@ \subsection{Mining rate and staking time}
...
-\section{Mining strategies and expected compensation}
+\section{Staking strategies and expected compensation}
 
 In this section, we look at three possibilities:
-a miner liquidating all the compensation while extending the lock time (Stake and Earn),
-a miner adding all the compensation to their current stake (Restake),
-and a miner waiting for their stake to unlock after time $T$ (Spindown).
+a staker liquidating all the compensation while extending the lock time (Stake and Earn),
+a staker adding all the compensation to their current stake (Restake),
+and a staker waiting for their stake to unlock after time $T$ (Spindown).
```
Comment (derekpierre) [inline mining-paper.tex]: Renamed to wind down:

```suggestion
and a staker waiting for their stake to unlock after time $T$ (Wind down).
```

File: mining-paper.tex [hunk 3]
```diff
@@ -266,21 +262,21 @@ \subsection{Restake mining compensation}
     \label{fig:total-compensation}
 \end{figure}
 
-\subsection{Take mining compensation and spindown}
+\subsection{Take compensation and spindown}
```
Comment (derekpierre) [inline mining-paper.tex]:
```suggestion
\subsection{Take compensation and wind down}
```

File: mining-paper.tex [hunk 4]
```diff
@@ -266,21 +262,21 @@ \subsection{Restake mining compensation}
...
 
-When the node spins down, the miner doesn't extend the time for end of staking $t_1$,
+When the node spins down, the staker doesn't extend the time for end of staking $t_1$,
```
Comment (derekpierre) [inline mining-paper.tex]:
```suggestion
When the node winds down, the staker doesn't extend the time for end of staking $t_1$,
```

File: mining-paper.tex [hunk 5]
```diff
@@ -266,21 +262,21 @@ \subsection{Restake mining compensation}
...
 and the compensation is constantly decreasing as the time left to unlock becomes smaller and smaller,
 effectively decreasing $\kappa$ gradually towards $0.5$.
-That's the default behavior: to avoid that, the miner should set $t_1$ large enough, or increase $t_1$ periodically.
+That's the default behavior: to avoid that, the staker should set $t_1$ large enough, or increase $t_1$ periodically.
```
Comment (derekpierre) [inline mining-paper.tex]: This is no longer the default...The default is that wind down is disabled.

File: mining-paper.tex [hunk 6]
```diff
@@ -266,21 +262,21 @@ \subsection{Restake mining compensation}
...
-If a miner is found to be non-operational and/or cannot confirm its continued performance, the tokens aren't unlocked (and compensation isn't earned).
-It's not fatal for the miner if that happens (i.e. their entire stake won't be slashed), however their tokens will remain locked without earning compensation,
+If a staker is found to be non-operational and/or cannot confirm its continued performance, the tokens aren't unlocked (and compensation isn't earned).
+It's not fatal for the staker if that happens (i.e. their entire stake won't be slashed), however their tokens will remain locked without earning compensation,
 until they satisfy their commitment.
-So if the miner commits to stake for at least a year, it implies a year of operation.
+So if the staker commits to stake for at least a year, it implies a year of operation.
 
-Connection problems may also be downtime when the miner upgrades their node's software, which is an absolutely legitimate reason to be offline for a short time.
+Connection problems may also be downtime when the staker upgrades their node's software, which is an absolutely legitimate reason to be offline for a short time.
```
Comment (derekpierre) [inline mining-paper.tex]:
```suggestion
Connection problems may include downtime when the staker upgrades their node's software, which is a legitimate reason to be offline for a short time.
```

File: mining-paper.tex [hunk 7]
```diff
@@ -296,14 +292,14 @@ \subsection{How much will I be compensated if I run a node?}
...
 
 \subsection{How many tokens will ever be in existence?}
-We'll start with $1$~billion tokens, and the maximum amount of tokens ever mined will be $3.89$~billion.
+We'll start with $1$~billion tokens, and the maximum amount of tokens ever minted and distributed will be $3.89$~billion.
 
 \subsection{What's the inflation rate?}
-The inflation rate will depend on how many short-term miners and long-term miners are working in the system.
-Depending on this, the initial inflation will be between $50\%$~APR (if all miners are very short term) and $100\%$~APR (if all miners commit for a long
+The inflation rate will depend on how many short-term stakers and long-term stakers are working in the system.
+Depending on this, the initial inflation will be between $50\%$~APR (if all stakers are very short term) and $100\%$~APR (if all stakers commit for a long
```
Comment (derekpierre) [inline mining-paper.tex]: For this statement and the next, is it a question of **all stakers are...** or **a sufficient number/majority of stakers are...**?

File: mining-paper.tex [hunk 8]
```diff
@@ -26,40 +26,36 @@
...
 
 \date{\today}
 \maketitle
 
 \section{Motivation}
 
-Upon deployment of the \nucypher~network, and as the number of developers and applications using \nucypher~grows, payment for re-encryption services (`network fees') will account for an increasing proportion of earnings made by network nodes.
-However, before developer adoption has reached a sufficient threshold, nodes must be incentivized via a block compensation subsidy.
-This can be achieved via an inflation schedule that is distributed exclusively to miners.
+As adoption of the \nucypher~network increases, payments for access control services (`fees') will account for an increasing proportion of the earnings made by network nodes. However, before adoption has reached a sufficient threshold, nodes must be incentivized and subsidized via a regular inflationary reward.
```
Comment (mswilkison) [inline mining-paper.tex]:
```suggestion
If adoption of the \nucypher~network increases, payments for access control services (`fees') may account for an increasing proportion of the earnings made by network nodes. However, before adoption has reached a sufficient threshold, nodes must be incentivized and subsidized via a regular inflationary reward.
```

File: mining-paper.tex [hunk 9]
```diff
@@ -26,40 +26,36 @@
...
-    \item Periods of high inflation do not significantly depreciate the token price, maintaining liquidity for new stakers;
-    \item Stakers are strongly incentivized to stay online all the time, in order to maximize availability to users requiring re-encryption services;
-    \item Stakers are incentivized to `re-stake' inflation compensation in order to maximize their long-term compensation;
-    \item Short-term staking is also encouraged and subsidized, albeit at a lower rate.
+    \item The amount of rewards distributed to stakers is proportional to the relative size of their stake;
+    \item The amount of work assigned to stakers (and, generally, the sum of fees earned) is proportional to the relative size of their stake;
+    \item Stakers are incentivized to stake for longer durations by a commensurately higher reward rate;
+    \item Short-term staking is also encouraged, albeit at a lower reward rate;
+    \item Epochs with higher levels of inflation do not significantly depreciate the native token price;
+    \item Stakers are incentivized to `re-stake' rewards in order to maximize their long-term compensation.
```
Comment (mswilkison) [inline mining-paper.tex]:
```suggestion
    \item Stakers are incentivized to `re-stake' rewards in order to maximize their long-term stake in the network.
```

File: mining-paper.tex [hunk 10]
```diff
@@ -106,11 +102,11 @@ \section{Mining protocol}
     \label{fig:mining-modes}
 \end{figure}
 
-At any point, a miner can split their stake or any piece of their stake into two pieces~(\figref{fig:mining-modes}).
+At any point, a staker can split their stake or any piece of their stake into two pieces~(\figref{fig:mining-modes}).
 The size of each piece should be not less than $S_{\min}$.
-The reason miners may want to split their stake is because their compensation rate depends on the lock time $T$,
+The reason stakers may want to split their stake is because their compensation rate depends on the lock time $T$,
```
[the remark this answers is not in the document]
Comment (mswilkison) [reply mining-paper.tex]: Agree that this is the underlying reason.

File: mining-paper.tex [hunk 11]
```diff
@@ -129,8 +125,8 @@ \subsection{Initial inflation}
 
 \subsection{Inflation decay}
 
-Initially, inflation subsidizes mining, but payments for re-encryption services will generate the majority or all of the revenues of miners in the long run.
-If all miners have the same, maximum compensation rate, we choose the inflation rate to decay by factor of $2$ in $T_{1/2} = 2$ years.
+Initially, inflation subsidizes staking, but payments for re-encryption services will generate the majority or all of the revenues earned by stakers in the long run.
```
Comment (mswilkison) [inline mining-paper.tex]:
```suggestion
Initially, inflation subsidizes staking, but payments for re-encryption services may generate the majority or all of the revenues earned by stakers in the long run as the inflation subsidy eventually runs out.
```

File: mining-paper.tex [hunk 12]
```diff
@@ -26,40 +26,36 @@
...
-    \item The amount of work assigned to stakers (and, hence, compensation earned) is proportional to their stake;
-    \item Stakers are incentivized (by a higher compensation rate) to run nodes for longer periods;
-    \item Periods of high inflation do not significantly depreciate the token price, maintaining liquidity for new stakers;
-    \item Stakers are strongly incentivized to stay online all the time, in order to maximize availability to users requiring re-encryption services;
-    \item Stakers are incentivized to `re-stake' inflation compensation in order to maximize their long-term compensation;
-    \item Short-term staking is also encouraged and subsidized, albeit at a lower rate.
+    \item The amount of rewards distributed to stakers is proportional to the relative size of their stake;
+    \item The amount of work assigned to stakers (and, generally, the sum of fees earned) is proportional to the relative size of their stake;
+    \item Stakers are incentivized to stake for longer durations by a commensurately higher reward rate;
+    \item Short-term staking is also encouraged, albeit at a lower reward rate;
```
[the remark this answers is not in the document]
Comment (arjunhassard) [reply mining-paper.tex]:
```suggestion
    \item Short-term staking is acceptable, albeit at a lower reward rate;
```

File: mining-paper.tex [hunk 13]
```diff
@@ -106,11 +102,11 @@ \section{Mining protocol}
     \label{fig:mining-modes}
 \end{figure}
 
-At any point, a miner can split their stake or any piece of their stake into two pieces~(\figref{fig:mining-modes}).
+At any point, a staker can split their stake or any piece of their stake into two pieces~(\figref{fig:mining-modes}).
 The size of each piece should be not less than $S_{\min}$.
-The reason miners may want to split their stake is because their compensation rate depends on the lock time $T$,
+The reason stakers may want to split their stake is because their compensation rate depends on the lock time $T$,
 which will be discussed in more details later.
```
Comment (arjunhassard) [inline mining-paper.tex]:
```suggestion
which will be discussed in more detail later.
```

File: mining-paper.tex (hunk 5 above)
Comment (arjunhassard) [reply mining-paper.tex]:
```suggestion
The default setting for all stakes is for $t_1$ to be automatically prolonged, which maintains the maximum reward rate ($\kappa$ = 1). Stakers wishing to withdraw must proactively initiate a wind down on some or all of their stake.
```

**Pull request merged**